env/gym-ww/gym_ww/envs/ww3_env.py

```diff
@@ -86,16 +86,46 @@ def scale_point(self,point):
     def step(self, action):
         action = self.scale_point(self.actions_dict[action])
         self.state = np.array([self.state[0]+action[0], self.state[1]+action[1]])
```

Comment:
Comment or remove old self.state line?

Reply:
Both of them needs to be there for now at least

`self.state = np.array([self.state[0]+action[0], self.state[1]+action[1]])
        self.state = np.clip(self.state,0,(self.grid_height-1)*self.distance_points) `

The first one changes the state, the second one makes it stay inside the boundary

Reply:
Ok! :)